raiden/tests/integration/api/test_restapi.py

```diff
@@ -1537,11 +1534,13 @@ def test_get_token_network_for_token(
 ):
     app0 = raiden_network[0]
 
-    new_token_address = deploy_contract_web3(
-        CONTRACT_HUMAN_STANDARD_TOKEN,
-        app0.raiden.rpc_client,
-        contract_manager=contract_manager,
-        constructor_arguments=(token_amount, 2, "raiden", "Rd"),
+    new_token_address = TokenAddress(
```

Comment:
Why not create a helper function for this deployment which also returns the proper type?